docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -148,6 +148,12 @@ Offset identifier to start pagination from as returned by the `next` field in th
 Numeric offset to start pagination from based on the snapshots matching this request. Using a non-zero value for this parameter is mutually
 exclusive with using the `after` parameter. Defaults to `0`.
 
+`slm_policy`::
+(Optional, string)
+Filter snapshots by a comma-separated list of SLM policy names that snapshots belong to. Also accepts wildcards (`\*`) and combinations
```

Comment:
Had to escape this for it to render properly, not sure this is the right way as I don't fully understand why I had to escape here in particular when we never escape these anywhere else?

Reply:
Maybe @jrodewig can help out on that part?

Reply:
I believe you need to do that because there are more asterisks down the way in the sentence. Two asterisks denote a bold markup in Asciidoc. So if you don't escape the first asterisk, you will get a piece of text being rendered bold:
![Screenshot from 2021-09-07 10-59-47](https://user-images.githubusercontent.com/1717632/132316670-2a5dd43c-0d97-45c8-b968-661771e2d75e.png)